### ISSUE TITLE
Feature Request: Require "X-Content-Type-Options: nosniff" HTTP response header for alternate stylesheets

### ISSUE THREAD

**Issue by dgotilla** [webhintio/hint#1188]
I noticed Sonarwhal generates a Security Error when alternate stylesheets are served with the "X-Content-Type-Options: nosniff" HTTP response header (eg. "'x-content-type-options' header is not needed"). However, I see no reason alternate stylesheets should be treated differently than regular stylesheets; the MIME-type confusion attack vector is the same as browsers (or extensions) may allow users to enable alternate stylesheets at their discretion.

In other words, I believe the "X-Content-Type-Options: nosniff" HTTP header _should_ be required of both stylesheets and alternate stylesheets and alternate stylesheets served with this header _should not_ trigger the "header is not needed" Security Error.

**Comment by alrra** [comment]
> I see no reason alternate stylesheets should be treated differently than regular stylesheets

@dgotilla Yes, that is true. We should update the code.
